autotest/utils/run_client_chat.py

```diff
@@ -104,7 +104,7 @@ def command_test(config,
         file.writelines('reproduce command chat: ' + ' '.join(cmd) + '\n')
 
         spliter = '\n\n'
-        if 'codellama' in model.lower() and ' chat ' in cmd:
+        if 'codellama' in model.lower() and 'serve' not in ' '.join(cmd):
```

Comment:
why there is a special case about "'serve' not in ' '.join(cmd)"

Reply:
> why there is a special case about "'serve' not in ' '.join(cmd)"

currently, if use lmdeploy chat $codellama_model, use enter !! to end the input >>>
if use lmdeploy serve api_server http://localhost:23333 after apt_server is started, use double enter to end input >>>

I use this conditional statement to distinguish two situations. I did not notice cmd is a string list, fix this condition statement. Both 'chat' in ' '.join(cmd) and 'serve' not in ' '.join(cmd) are OK.